src/resmom/catch_child.c

```diff
@@ -106,14 +106,14 @@ extern int		server_stream;
 extern time_t		time_now;
 extern pbs_list_head	mom_polljobs;
 extern unsigned int	pbs_mom_port;
-extern int secondary_pbs_nodefile;
+extern int 		gen_nodefile_on_sister_mom;
```

Comment:
per our updated coding standards, variables and declarations need not be aligned - so you need to remove the alignment of all the blocks u touch